index.html

```diff
@@ -5,11 +5,377 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Medical care</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="/css/index.css" />
   </head>
   <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+    <header class="header">
+      <nav class="nav">
+        <h5 class="nav__logo">Medical Care</h5>
+        <ul class="nav__list">
+          <li class="nav__item"><a href="/#" class="nav__link">Home</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Products</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Pricing</a></li>
+          <li class="nav__item"><a href="/#" class="nav__link">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="header__action">
+        <button class="button button--primary button--visibility-default">
+          Login
+        </button>
+        <button
+          class="button button--secondary button--visibility-default button--after-icon-arrow"
+        >
+          Join Us
+        </button>
+        <button class="button button--visibility-mobile header__menu-icon">
```

Comment:
This brings about another issue. A block cannot have an element class in combination, making `header__menu-icon` invalid within this `button` block.